lsp/markdown/completion/configGet.md

```diff
@@ -0,0 +1,9 @@
+Snippet for getting a config variable.
+
+```go
+username = defaultUser.Get(ctx)
+```
+
+See https://tbd54566975.github.io/ftl/docs/reference/secretsconfig/
+---
+${1:value} := ${2:configVar}.Get(ctx)
```

Comment:
I'm not sure these snippets add much value, WDYT? Normal completion on the config var will show people what to do? Though I guess it can't hurt?

Reply:
I did consider that. Yeah it'll make LSP maintenance lower for us too.